CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Features
 
 - Use `recordHttpBreadcrumbs` to set iOS `enableNetworkBreadcrumbs` ([#1884](https://github.com/getsentry/sentry-dart/pull/1884))
+- Apply `beforeBreadcrumb` on native iOS crumbs ([#1914](https://github.com/getsentry/sentry-dart/pull/1914))
```

Comment:
I think it should be `native breadcrumbs` since it's not only for iOS? correct me if I'm wrong

Reply:
This does not work on Android afaik, only iOS/macOS

Reply:
ah yes my bad, the contexts integration is cocoa specific, nvm